src/slices/azureSlice.js

```diff
@@ -179,49 +177,65 @@ export const importRelease = (
   params,
   id,
   title
-) => async dispatch => {
+) => async (dispatch) => {
   dispatch(importReleasePending());
-  try {
-    // Get Ids of work items
-    fetchWorkItemIds(project, params, id).then(res => {
+
+  // Get Ids of work items
+  fetchWorkItemIds(project, params, id)
+    .then((res) => {
       var ids = res.data.value.map(({ id }) => id);
 
       // Get data of each work item
-      fetchWorkItems(project, params, ids).then(res => {
-        var rawWorkItems = res.data.value;
-        var workItems = [];
-
-        // Format and filter each work item
-        rawWorkItems.forEach(wi => {
-          workItems.push(createWorkItem(wi));
+      fetchWorkItems(project, params, ids)
+        .then((res) => {
+          var rawWorkItems = res.data.value;
+          var workItems = [];
+
+          // Format and filter each work item
+          rawWorkItems.forEach((wi) => {
+            workItems.push(createWorkItem(wi));
+          });
+
+          // Create new release
+          return {
+            title: title,
+            isPublic: false,
+            productVersionId: productVersionId,
+            releaseNotes: workItems,
+          };
+        })
+        .then((release) => {
+          // post new release
+          dispatch(importReleaseSuccess());
+          return release;
```

Comment:
Klar ikke helt å se kor du return, treng du den linja der?

Reply:
returna release her for å chaine den til neste `then` blokk

Reply:
Når eg ser nærmere på det, så skjønn eg ikke koffer `dispatch(importReleaseSuccess());` og `dispatch(postRelease(release))` e lagt i forskjellige `.then()`

Reply:
Sant. Hensikta med det gikk litt tapt. Fixed

Reply:
Det går bra, pappa e glad i deg